lib/modules/datasource/npm/yarnrc.ts

```diff
@@ -0,0 +1,46 @@
+import { load } from 'js-yaml';
+import { z } from 'zod';
+import { logger } from '../../../logger';
+
+const YarnrcYmlSchema = z.object({
```

Comment:
did you validate that other yarn config properties are allowed? eg linker ?

Reply:
Interactively yes, but now I added a test case to ensure we don't break this!